gnd/src/main/java/com/google/android/gnd/repository/local/Edit.java

```diff
@@ -61,17 +65,30 @@ public static int toInt(@Nullable Edit.Type value) {
   public abstract Type getType();
 
   /**
-   * For edits of type UPDATE, returns a JSON object with the original value of modified attributes
-   * before this edit. For all other edit types this will return null.
+   * For edits of type {@link Edit.Type.UPDATE}, returns a JSON object with the original values of
```

Comment:
I *think* the correct way to specify the `@link` target here is:
```java
{@link Type#UPDATE}
```
since `UPDATE` is a (static) field in a different class, and `Edit` isn't necessary since we are already in the same class. Check the link syntax and let me know if you come to the same conclusion: https://docs.oracle.com/javase/8/docs/technotes/tools/unix/javadoc.html#JSSOR654

Also `old_values` will be set on type `DELETE` edits as well, correct?

Reply:
Done. Thanks!

Also, I believe we would in fact need to keep old values on DELETE in order to allow rollback. Good catch.